site/_docs/avatica_roadmap.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Roadmap
```

Comment:
Oh I see why I did this now. Definitely want the title to be "Avatica Roadmap" but having "Avatica Roadmap" under the "Avatica" section in the sidebar is duplicative IMO. Let me see if I can work around that.